CSharp/core-DirectLineWebSockets/DirectLineClient/Program.cs

```diff
@@ -0,0 +1,118 @@
+﻿namespace DirectLineSampleClient
+{
+    using System;
+    using System.Configuration;
+    using System.Diagnostics;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using Microsoft.Bot.Connector.DirectLine;
+    using Newtonsoft.Json;
+    using System.Threading;
+    using System.Text;
+    using WebSocketSharp;
+
+    public class Program
+    {
+        private static string directLineSecret = ConfigurationManager.AppSettings["DirectLineSecret"];
+        private static string botId = ConfigurationManager.AppSettings["BotId"];
+        private static string fromUser = "DirectLineSampleClientUser";
```

Comment:
Can you add the following comment above fromUser?

// fromUser is the field that identifies which user is sending activities to the Direct Line service.
// Because this value is created and sent within your Direct Line client, your bot should not
// trust the value for any security-sensitive operations. Instead, have the user log in and
// store any sign-in tokens against the Conversation or Private state fields. Those fields
// are secured by the conversation ID, which is protected with a signature.